### DISCUSSION TITLE
save row selected with radio button  in Repository.cs.?

### DISCUSSION THREAD

**Discussion by hosnabigdeli** [serenity-is/Serenity#5722] [Help]
hi 
i have master/detail  form and in detail i have  several radio button i want to save row id was selected in Repository.cs.

**Comment by minhhungit** [comment]
I suggest you should not implement that way, that is **very bad UX** also it will leak many problems, you should find another way without using detail grid or implementing yourself editor (grid detail is also an editor) that support radio selector and return only one row
